### DISCUSSION TITLE
FreeBSD support

### DISCUSSION THREAD

**Discussion by avalx** [jcorporation/myMPD#1154] [Q&A] [answered]
I was very glad to find out that FreeBSD is going to be supported as I have a server running FreeBSD for a long time. And I'm already using myMPD on my other system and it is the best way to control MPD instances!

So I tried to install myMPD as per instructions but after `make install` I get some errors and the process does not finish (errors in the attachment). I've seen the info on the Installation instruction page that says: "full support since FreeBSD 14"! At the moment I have the 13.2-RELEASE-p4 version so does this mean that I'll have to wait for 14 release to be able to install myMPD?

 Thanks again for all the efforts working on this project.

[e.txt](https://github.com/jcorporation/myMPD/files/13248308/e.txt)

**Comment by jcorporation** [comment, marked as answer]
This is a bug that is already fixed in devel. Try the devel branch or wait for the v13.0.2 myMPD release.

With FreeBSD < 14 only timers are not supported.

**Comment by avalx** [comment]
Great, I'll wait for the new release.
